deploy/openshift/admin-elasticsearch-template.yaml

```diff
@@ -20,7 +20,7 @@ objects:
       plural: elasticsearches
       singular: elasticsearch
     scope: Namespaced
-    version: v1alpha1
+    version: v1
```

Comment:
side note, not sure why we still have these files in the repo... todo in future?

Reply:
yes... there are a bunch of these I'm not sure if/where they are used.  they all really need to be condensed and consolidated to only use whatever ends up in the manifest